app/controllers/manager_profiles_controller.rb

```diff
@@ -36,6 +37,11 @@ def remove_staff
   end
 
   def manage_staffs
+    @roles = {}
+    EventStaff.roles.each do |key, value|
+       role_text = key.split("_").map(&:capitalize).join(" ")
+       @roles[role_text] = key
+     end
```

Comment:
`end` at 44, 5 is not aligned with `EventStaff.roles.each do |key, value|` at 41, 4.